Automattic/wp-calypso
# Add ectoplasm color scheme (Method A)

One part of #47036
Method A is defined in #47103. There's a "Method B" copy of the PR here: #47075.

#### Changes proposed in this Pull Request

* Add ectoplasm color scheme

#### Testing instructions

* Visit /me/account
* Choose ectoplasm and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-04_15-54](https://user-images.githubusercontent.com/937354/98171825-fdbf0400-1eb5-11eb-82c6-17af95880050.png)
Wp-Admin:
![2020-11-04_15-54_1](https://user-images.githubusercontent.com/937354/98171911-15968800-1eb6-11eb-95f3-c7cc1a6f21d7.png)

#### Known Issues

* Big: All lightness values of both primary and accent colors, -10 through -90, are set to the same color.  How do we generate these while maintaining a seamless experience between wp-admin and calypso?  [Here's how color-studio generates them](https://github.com/Automattic/color-studio/blob/master/data/color-definitions.js).
* Primary and accent colors are the same.  Does this cause an issue?
